`riscv64` does not support `LuaJIT` :

```
  lj_arch.h:73:2: error: #error "No support for this architecture (yet)"
     73 | #error "No support for this architecture (yet)"
```

--- a/app-utils/yazi/autobuild/beyond
+++ b/app-utils/yazi/autobuild/beyond
@@ -1,0 +1,12 @@
+abinfo "Installing completions ..."
+mkdir -pv "$PKGDIR"/usr/share/bash-completion/completions
+cp -v "$SRCDIR"/config/completions/yazi.bash \
+        "$PKGDIR"/usr/share/bash-completion/completions/yazi.bash
+
+mkdir -pv "$PKGDIR"/usr/share/fish/completions
+cp -v "$SRCDIR"/config/completions/yazi.fish \
+        "$PKGDIR"/usr/share/fish/completions/yazi.fish
+
+mkdir -pv "$PKGDIR"/usr/share/zsh/site-functions
+cp -v "$SRCDIR"/config/completions/_yazi \
+	"$PKGDIR"/usr/share/zsh/site-functions/_yazi

--- a/app-utils/yazi/autobuild/defines
+++ b/app-utils/yazi/autobuild/defines
@@ -1,0 +1,10 @@
+PKGNAME="yazi"
+PKGDES="A file manager for the terminal"
+PKGDEP="glibc gcc-runtime jq"
+BUILDDEP="rustc"
+PKGSEC=utils
+
+ABSPLITDBG=0
+
+# FIXME: mlua-sys must be compiled with gcc
+NOLTO=1

--- a/app-utils/yazi/autobuild/patches/0001-fix-compile.patch
+++ b/app-utils/yazi/autobuild/patches/0001-fix-compile.patch
@@ -1,0 +1,51 @@
+From 882b54d5a9a26720bdc0e74590aa52f2b23f4887 Mon Sep 17 00:00:00 2001
+From: eatradish <sakiiily@aosc.io>
+Date: Thu, 7 Dec 2023 01:25:58 -0800
+Subject: [PATCH] fix compile
+
+---
+ core/src/tab/mod.rs        | 2 +-
+ plugin/src/bindings/mod.rs | 2 +-
+ plugin/src/lib.rs          | 2 +-
+ 3 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/core/src/tab/mod.rs b/core/src/tab/mod.rs
+index ebcd998..422995e 100644
+--- a/core/src/tab/mod.rs
++++ b/core/src/tab/mod.rs
+@@ -7,7 +7,7 @@ mod mode;
+ mod tab;
+ 
+ pub use backstack::*;
+-pub use config::*;
++pub use self::config::*;
+ pub use finder::*;
+ pub use folder::*;
+ pub use mode::*;
+diff --git a/plugin/src/bindings/mod.rs b/plugin/src/bindings/mod.rs
+index 60bbb91..d419dcc 100644
+--- a/plugin/src/bindings/mod.rs
++++ b/plugin/src/bindings/mod.rs
+@@ -10,6 +10,6 @@ mod tasks;
+ pub use active::*;
+ pub use bindings::*;
+ pub use files::*;
+-pub use shared::*;
++pub use self::shared::*;
+ pub use tabs::*;
+ pub use tasks::*;
+diff --git a/plugin/src/lib.rs b/plugin/src/lib.rs
+index b383da4..286a96e 100644
+--- a/plugin/src/lib.rs
++++ b/plugin/src/lib.rs
+@@ -8,6 +8,6 @@ mod plugin;
+ mod scope;
+ mod utils;
+ 
+-use config::*;
++use self::config::*;
+ pub use plugin::*;
+ pub use scope::*;
+-- 
+2.39.1
+

--- a/app-utils/yazi/spec
+++ b/app-utils/yazi/spec
@@ -1,0 +1,4 @@
+VER=0.1.5
+SRCS="git::commit=tags/v$VER::https://github.com/sxyazi/yazi"
+CHKSUMS="SKIP"
+CHKUPDATE="anitya::id=370571"

